Sources/SwerverCore/Request/RequestMethod.swift

```diff
@@ -1,6 +1,17 @@
 public enum RequestMethod: String {
     case get = "GET"
+    case post = "POST"
+    case put = "PUT"
+    case patch = "PATCH"
+    case delete = "DELETE"
+    case copy = "COPY"
     case head = "HEAD"
     case options = "OPTIONS"
-    case other = "OTHER"
+    case link = "LINK"
```

Comment:
Have you implemented all of these verbs? Do you need to include them all here?

Reply:
this is for request parser to identified valid http request methods so that when it receives a request string, it won't mistake something from being invalid. For example, when a request comes in with a method other than those listed in the enum, it'll throw and error because the enum RequestMethod(rawValue: String) returns nil. If I don't include these in the enum and the server receives a POST request, that'll be considered an invalid when it should have returned a 405 method not allowed response.